caffeine@patapon.info/extension.js

```diff
@@ -158,7 +157,7 @@ class CaffeineToggle extends QuickSettings.QuickMenuToggle {
             // The 'label' property was renamed to 'title' in GNOME 44 but quick settings have otherwise 
             // not been changed. The below line allows support for both GNOME 43 and 44+ by using the 
             // appropriate property name based on the GNOME version.
-            [ShellVersion >= 44 ? 'title' : 'label']: IndicatorName,
```

Comment:
This still leaves an instance of IndicatorName, which is then untranslated. Was this intentional? If the goal was to do the same thing as `TimerMenuName`, then I don't understand the reasoning, for the same reason as my last comment.

Reply:
Unless the intention was to not translate the remaining instance of `IndicatorName`, but if that's the case, I think the constant should probably be removed, as there's only 1 instance of it left, and it's an unchanging string.

Reply:
`IndicatorName` is still used as a generic name for the inhibit flag only (so it will stay untranslated):
```
addInhibit(appId) {
        this._sessionManager.InhibitRemote(appId,
            0, 'Inhibit by %s'.format(IndicatorName), this.inhibitFlags,
            cookie => {
```
We can removed this constant and maybe we could also get this name from the json file with `Me.metadata.name` instead.

Reply:
If it doesn't need to be translated, why can't we just hard-code it to say 'Inhibit by Caffeine'?

Reply:
I personally prefer to avoid hard coding, but I'm maybe be old school. The name of the extension is define in the metadata.json so I reckon it would be cleanest to use this instead.

Reply:
That's understandable, but since the code is for this extension and not a generic piece of code dropped into multiple extensions, it makes sense to hard-code the string, instead of just querying the extensions system to retrieve another hard-coded string.

Ultimately, if it's cleanly done I'm not really opposed to it, I started this comment thread to try get at the reasoning for some of the changes, that's all :)